src/hooks/useFarmStats.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react'
+
+import gql from 'graphql-tag'
+import { Client } from 'urql'
+
+import { WPOKT_SUBGRAPH_URL } from 'util/constants'
+
+const RETRY_EVERY = 3000
+
+const graphqlClient = new Client({ url: WPOKT_SUBGRAPH_URL })
+
+export function useFarmStats(farmAddress) {
+  const [totalStaked, setTotalStaked] = useState(0)
+  const [apy, setAPY] = useState(0)
+  const [tvl, setTVL] = useState(0)
+
+  const FARM_STATS_QUERY = gql`
+    query {
+      tokenGeysers(id: "${farmAddress}") {
+        staked
+        apy
+        tvl
+      }
+    }
+  `
```

Comment:
I think we can declare the query outside the hook, for if we ever need it. What do you think?